jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBConnection.java

```diff
@@ -89,7 +90,10 @@ public IoTDBConnection(String url, Properties info) throws SQLException, TTransp
     supportedProtocols.add(TSProtocolVersion.TSFILE_SERVICE_PROTOCOL_V1);
 
     openTransport();
-    client = new TSIService.Client(new TBinaryProtocol(transport));
+    if(Config.rpcThriftCompressionEnable)
+      client = new TSIService.Client(new TCompactProtocol(transport));
+    else
+      client = new TSIService.Client(new TBinaryProtocol(transport));
```

Comment:
Please use brackets even for a single statement in if or for.

Reply:
Alright, thanks for your suggestion.